pycbc/tmpltbank/coord_utils.py

```diff
@@ -291,7 +291,9 @@ def get_random_mass(numPoints, massRangeParams):
 
     return mass1, mass2, spin1z, spin2z
 
-def get_cov_params(mass1, mass2, spin1z, spin2z, metricParams, fUpper):
+def get_cov_params(mass1, mass2, spin1z, spin2z, metricParams, fUpper,
```

Comment:
This function now has many arguments. Kwargs for `lambda*` and `quadparam*` would help, or maybe even a record array carrying all parameters at once?

Reply:
I might look to refactor this to take some form of `mass_params` structure in the future, but not today.